### PR TITLE
DL-2982:Added css files to correct colour contrasts for WCAG 2.1

### PR DESCRIPTION
**Pull request by nadinJ**
DL-2982:Added css files to correct colour contrasts for WCAG 2.1

Bug fix 

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist Reviewee (add name here)

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date
 
 ## Checklist Reviewer (Paul Anderson)
 
  - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
  - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
  - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
  - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
  - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: project/FrontendBuild.scala [hunk 1]
```diff
@@ -10,19 +10,19 @@ object FrontendBuild extends Build with MicroService {
     import play.sbt.PlayImport.ws
     val compile: Seq[ModuleID] = Seq(
       ws,
-      "uk.gov.hmrc" %% "bootstrap-play-26" % "1.4.0",
+      "uk.gov.hmrc" %% "bootstrap-play-26" % "1.7.0",
       "uk.gov.hmrc" %% "domain" % "5.6.0-play-26",
       "uk.gov.hmrc" %% "play-partials" % "6.9.0-play-26",
```
Comment (paulanderson1234) [inline project/FrontendBuild.scala]: This (play-partials) can go up to version 6.10.0-play-26 I think

File: project/FrontendBuild.scala [hunk 2]
```diff
@@ -10,19 +10,19 @@ object FrontendBuild extends Build with MicroService {
...
       "uk.gov.hmrc" %% "play-ui" % "8.8.0-play-26",
       "uk.gov.hmrc" %% "http-caching-client" % "9.0.0-play-26",
-      "uk.gov.hmrc" %% "auth-client" % "2.33.0-play-26",
-      "com.typesafe.play" %% "play-json-joda" % "2.6.13",
-      "uk.gov.hmrc" %% "govuk-template" % "5.52.0-play-26"
+      "uk.gov.hmrc" %% "auth-client" % "2.35.0-play-26",
```
Comment (paulanderson1234) [inline project/FrontendBuild.scala]: auth-client can probably go to 3.0.0-play-26 ...

File: app/views/details_not_found.scala.html [hunk 3]
```diff
@@ -51,6 +51,9 @@ <h2 class="heading-small" id="details-not-found-name-changed-reason-header">@mes
   <h2 class="heading-small" id="details-not-found-utr-reason-header">@messages("bc.details-not-found.utr-reason.header")</h2>
   <p id="details-not-found-utr-reason-text">@messages("bc.details-not-found.utr-reason.text")</p>
 
-  <a class="button" role="button" data-journey-click="business-customer:click:try-matching-again" href="@controllers.routes.BusinessVerificationController.businessForm(service, businessType)">@messages("bc.details-not-found.try-again")</a>
+
+@form(action=controllers.routes.BusinessVerificationController.businessForm(service, businessType)){
+<button class="button" role="button" data-journey-click="business-customer:click:try-matching-again" >@messages("bc.details-not-found.try-again")</button>
```
Comment (paulanderson1234) [inline app/views/details_not_found.scala.html]: We probably need to remove all instances of role="button" as it is observed in the VNU log https://build.tax.service.gov.uk/job/DDCT%20Live%20Services/job/Business%20Customer%20Services/job/business-customer-accessibility-tests/2/Accessibility_20Check_20Report/

Comment (nadinJ) [reply app/views/details_not_found.scala.html]: I will make all the changes and run the tests again

**Pull request merged**